src/OperationalInsights/OperationalInsights/LinkedStorageAccount/NewAzureOperationalInsightsLinkedStorageAccountCommand.cs

```diff
@@ -36,11 +37,12 @@ public class NewAzureOperationalInsightsLinkedStorageAccountCommand : Operationa
 
         [Parameter(Position = 2,
             Mandatory = true,
-            HelpMessage = "Data Source Type should be one of 'CustomLogs', 'AzureWatson'.")]
+            HelpMessage = "Data Source Type should be one of 'CustomLogs', 'AzureWatson', 'Query', 'Alerts'.")]
         [ValidateNotNullOrEmpty]
-        [ValidateSet("CustomLogs", "AzureWatson")]
+        [ValidateSet("CustomLogs", "AzureWatson", "Query", "Alerts", IgnoreCase = true)]
         public string DataSourceType { get; set; }
 
+        [CmdletParameterBreakingChange("StorageAccountId")]
```

Comment:
```suggestion
```
Since renaming this parameter to plural might not be a good idea, we might just remove this line completely, as there is not going to be breaking change.